### DISCUSSION TITLE
Favourite type of music

### DISCUSSION THREAD

**Discussion by angelospl** [xpirit-training/github-fundamentals-signaloid#11] [General]
What is your favourite type of music and why is it metal?

**Comment by btsouts** [comment]
Why is it metal?

**Comment by almidi** [comment]
I like the apparel

**Reply by orestiskapar** [reply]
Apparel?

**Reply by almidi** [reply]
T-shirts with heavy metal band logos

**Comment by orestiskapar** [comment]
I just search youtube for "extreme coding music" and listen to it all day long

**Comment by bilgesubilgin-signaloid** [comment]
Classical or psychedelic stuff
